src/pages/PullRequestPage/hooks/usePullPageData.tsx

```diff
@@ -49,12 +49,18 @@ const RepositorySchema = z.object({
 })
 
 const PullPageDataSchema = z.object({
-  isCurrentUserPartOfOrg: z.boolean(),
-  repository: z.discriminatedUnion('__typename', [
-    RepositorySchema,
-    RepoNotFoundErrorSchema,
-    RepoOwnerNotActivatedErrorSchema,
-  ]),
+  owner: z
+    .object({
+      isCurrentUserPartOfOrg: z.boolean(),
+      repository: z
+        .discriminatedUnion('__typename', [
+          RepositorySchema,
+          RepoNotFoundErrorSchema,
+          RepoOwnerNotActivatedErrorSchema,
+        ])
+        .nullable(),
+    })
+    .nullable(),
```

Comment:
do we plan to take out the queries to their own files?

Reply:
I don't see any benefit as to removing the queries as when they're only used in a single location. I also think it's somewhat of a bad practice in these instances to move a string to it's own file.